### DISCUSSION TITLE
Windows Integrated authentication

### DISCUSSION THREAD

**Discussion by carethik2k** [cefsharp/CefSharp#3918] [General]
Hi all,
Does CEFSharp support windows integrated authentication? I am trying to navigate to a website that redirects first to Azure AD. I am trying to implement WIA so that the user is not prompted to enter his/her user credentials as shown in the below screenshot.
![image](https://user-images.githubusercontent.com/425756/144469122-40e2f793-4719-45e6-b414-8eef671b10b7.png)

In the case of Firefox browser, there is a setting that controls Windows Integrated Authentication (below screenshot). Do we have a setting in CEFSharp to turn on similar functionality?
![image](https://user-images.githubusercontent.com/425756/144469388-d2403d35-2f92-481f-acb8-8b1fd6a71fcc.png)

Thanks in advance for your help!

**Comment by amaitland** [comment]
There is an upstream issue with NTLM authentication at the moment. 

https://bitbucket.org/chromiumembedded/cef/issues/3110/ntlm-authenentication-not-working

**Reply by amaitland** [reply]
You can try the --disable-request-handling-for-testing command line argument.

See https://github.com/cefsharp/CefSharp/wiki/General-Usage#cefsettings-and-browsersettings for example of setting command line args.

**Reply by carethik2k** [reply]
Thanks, Alex. I'll give that a shot and get back. Appreciate the quick response.
